Alicerce/Sources/Logging/Log+NodeLogDestination.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  Log+NodeLogDestination.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class NodeLogDestination : LogDestination {
+
+        private static let dispatchQueueLabel = "com.mindera.Alicerce.NodeLogDestination"
+        private static let defaultRequestTimeout: TimeInterval = 0
+
+        public private(set) var dispatchQueue: DispatchQueue
+        public private(set) var minLevel: Level
+        public private(set) var formatter: LogItemFormatter
+
+        public var logItemsSent = 0
+
+        private let serverURL: URL
+        private let urlSession: URLSession
+        private let requestTimeout: TimeInterval
+
+        //MARK:- lifecycle
+
+        public init(serverURL: URL,
+                    minLevel: Level = Log.Level.error,
+                    formatter: LogItemFormatter = Log.StringLogItemFormatter(),
+                    urlSession: URLSession = URLSession.shared,
+                    dispatchQueue: DispatchQueue = DispatchQueue(label: NodeLogDestination.dispatchQueueLabel),
+                    requestTimeout: TimeInterval = NodeLogDestination.defaultRequestTimeout) {
+
+            self.serverURL = serverURL
+            self.minLevel = minLevel
+            self.formatter = formatter
+            self.urlSession = urlSession
+            self.dispatchQueue = dispatchQueue
+            self.requestTimeout = requestTimeout
+        }
+
+        //MARK:- public methods
+
+        public func write(item: Item) {
+            weak var weakSelf = self
+            dispatchQueue.sync {
+                let formattedItem = formatter.format(logItem: item)
+                if let payloadData = formattedItem.data(using: .utf8) {
+                    send(payload: payloadData) { success in
+                        guard let strongSelf = weakSelf else { return }
+                        if success { strongSelf.logItemsSent += 1 }
+                    }
+                }
+            }
+        }
+
+        //MARK:- private methods
+
+        private func send(payload: Data, completion: @escaping (_ success: Bool) -> Void) {
+
+            var request = URLRequest(url: serverURL,
+                                     cachePolicy: .reloadIgnoringLocalAndRemoteCacheData,
+                                     timeoutInterval: requestTimeout)
+
+            // setup the request's method and headers
+
+            request.httpMethod = "POST"
+            request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+            request.addValue("application/json", forHTTPHeaderField: "Accept")
```

Comment:
Honestly, I don't agree. This is our small test NodeJS destination, no *generic* destination.